_includes/performance.html

```diff
@@ -0,0 +1,64 @@
+<div class="full-width-bg component">
+  <div class="grid-wrapper">
+    <div class="width-3-12 width-12-12-m img-vert-center">
+      <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-performance.svg" alt="Performance icon">
+      <img class="dark-only" src="{{site.baseurl}}/assets/images/performance/icon-performance-dark.svg" alt="Performance icon">
+    </div>
+    <div class="width-9-12 width-12-12-m">
+      <h2>The Quarkus Way</h2>
+      <p>Quarkus redefines Java development by shifting work to the build phase. This approach minimizes runtime dependencies, maximizes dead code elimination, and introduces clear metadata contracts, resulting in leaner and more efficient applications.</p>
+      <p>The Quarkus approach helps to "strip out" all code which is only necessary to validate the user model</p>
+      <ul>
+        <li>compile classes or apply bytecode enhancements</li>
+        <li>introspect in the model</li>
+        <li>parse of framework configuration files</li>
+        <li>discover extensions and services, or “apply auto-configuration"</li>
+      </ul>
+      <p>All that code is run only during the compilation phase; the classes don't even get loaded when  the app is started. This means the JIT can make much better optimisations, and spend less memory in doing so.</p>
+      <p>By performing these optimizations at build time, Quarkus ensures that the heavy lifting is done once, not at every startup, reducing memory usage and startup time.</p>
+      <p>This approach benefits both GraalVM native images and traditional HotSpot JVM deployments, leading to faster, smaller, and more resource-efficient Java applications.</p>
+    </div>
+    <div class="width-3-12 width-12-12-m img-vert-center">
+      <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-memory.svg" alt="Memory icon">
+      <img class="dark-only" src="{{site.baseurl}}/assets/images/performance/icon-memory-dark.svg" alt="Memory icon">
+    </div>
+    <div class="width-9-12 width-12-12-m">
+      <h2>Reduced Memory</h2>
```

Comment:
Should be h3 - it's part of the quarkus way, which actually is named" build-time principle.